<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>579</width>
    <height>472</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>10</y>
     <width>471</width>
     <height>61</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>90</y>
     <width>361</width>
     <height>341</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QPushButton" name="pushButton_4">
        <property name="text">
         <string>4</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_5">
        <property name="text">
         <string>5</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_6">
        <property name="text">
         <string>6</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="2" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QPushButton" name="pushButton_7">
        <property name="text">
         <string>7</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_8">
        <property name="text">
         <string>8</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_9">
        <property name="text">
         <string>9</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="3" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QPushButton" name="pushButton_10">
        <property name="text">
         <string>(</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_11">
        <property name="text">
         <string>)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_12">
        <property name="text">
         <string>0</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="pushButton">
        <property name="text">
         <string>1</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_2">
        <property name="text">
         <string>2</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_3">
        <property name="text">
         <string>3</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>90</y>
     <width>151</width>
     <height>351</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QPushButton" name="pushButton_17">
      <property name="text">
       <string>+</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_18">
      <property name="text">
       <string>-</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_19">
      <property name="text">
       <string>*</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_20">
      <property name="text">
       <string>/</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_16">
      <property name="text">
       <string>BACKSPACE</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_13">
      <property name="text">
       <string>sin</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_14">
      <property name="text">
       <string>cos</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_15">
      <property name="text">
       <string>tan</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="calc">
      <property name="text">
       <string>Calc</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
